llvm/include/llvm/ProfileData/InstrProf.h

```diff
@@ -494,6 +498,14 @@ class InstrProfSymtab {
   // - In MD5FuncMap: <MD5Hash(name), &F> for name in name-set
   Error addFuncWithName(Function &F, StringRef PGOFuncName);
 
+  // Add the vtable into the symbol table, by creating the following
+  // map entries:
+  // name-set = {PGOName} + {getCanonicalName(PGOName)} if the canonical name
```

Comment:
+ --> Union

'if ...is different ' can be removed because it is implied by union operation.

Reply:
done.